webdriver/conftest.py

```diff
@@ -13,8 +14,9 @@
 def _session(request):
     host = os.environ.get("WD_HOST", default_host)
     port = int(os.environ.get("WD_PORT", default_port))
+    capabilities = json.loads(os.environ.get("WD_CAPABILITIES", "{}"))
 
-    session = webdriver.Session(host, port)
+    session = webdriver.Session(host, port, desired_capabilities=capabilities)
```

Comment:
I guess wdclient needs to change `desired_capabilities` to be spec compatible at some point.